Added privacy policy page

govindumeesala opened the pull request:
# Title and Issue number 
<!-- Please make sure issue number is mention in Pull Request else PR will not be merged. -->

Title : Added privacy policy page to the website

Issue No. :133

Code Stack : React Material UI

Close #133 

# Video (mandatory)
<!--Please try to attach the working video of your new deployed project here -->

https://github.com/user-attachments/assets/34b9e546-7310-43c2-80ae-ab1b1f3484ce

# Checklist:

- [x] I have mentioned the issue number in my Pull Request.
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md` if its a new page/tech stack
- [x] I have gone through the  `contributing.md` file before contributing


<!-- [X] - put a cross/X inside [] to check the box -->
**Additional context (Mandatory )**
Added privacy policy page

***Are you contributing under any Open-source programme?***
<!--Mention it here-->
- [x] I'm a GSSOC-EXT contributor
- [x] I'm a HACKTOBERFEST contributor
       
@jeevan10017 please check this out

jeevan10017 reviewed (changes requested):
Link it in footer, already navbar has full of li

govindumeesala commented:
> Link it in footer, already navbar has full of li

Added to footer and removed from navbar as required 

![image](https://github.com/user-attachments/assets/fe8508ae-8a3b-4984-8025-3a771557fd59)

@jeevan10017  please check once
